framework/lib/components/createable-select-dropdown/custom-components.tsx

```diff
@@ -4,20 +4,40 @@ import { PlusSolid } from '@northlight/icons'
 import { Icon } from '../icon'
 import { Box } from '../box'
 import { Flex } from '../flex'
+import { Text } from '../text'
 
 export const customComponents = {
   Option: ({ children, ...props }: any) => (
-    <chakraComponents.Option { ...props }>
-      <>
-        { props.data.isCreation ? (
-          <Flex mr={ 3 } width={ 1.5 } mb={ 0.5 } justifyContent="center" alignItems="center">
-            <Icon mb="4px" as={ PlusSolid } color="brand" />
-          </Flex>
-        ) : (
-          <Box mr={ 3 } width={ 1.5 } />
-        ) }
-        { children }
-      </>
-    </chakraComponents.Option>
+    <Box
+      onClick={ () => {
+        if (props.data.isCreation) {
+          props.selectProps.toggleCreateNewOption()
+        }
+      } }
+    >
+      <chakraComponents.Option { ...props }>
+        <>
+          { props.data.isCreation ? (
+            <Flex
+              mr={ 3 }
+              width={ 1.5 }
+              mb={ 0.5 }
+              justifyContent="center"
+              alignItems="center"
+            >
+              <Icon as={ PlusSolid } color="brand" />
+            </Flex>
+          ) : (
+            <Box mr={ 3 } width={ 1.5 } />
+          ) }
+          <Text
+            color={ props.data.isCreation ? 'brand' : 'default' }
```

Comment:
Sorry for popping up uninvited. Can we use the system tokens here instead of the reference tokens:
`'text.brand'`
`'text.default'`

These tokens reflect decisions and describe intention/usage/quality. Please aim to use only system tokens. DM me for further explanation.